content/en/blog/2022-05-05-introducing-kluctl-and-flux.md

```diff
@@ -0,0 +1,83 @@
+but instead an essential tool and building block to make it work.
+
+Kluctl can be looked at from the same perspective when it comes to Flux. Flux implements Helm and Kustomize support
+via different controllers, namely the [kustomize-controller](https://fluxcd.io/docs/components/kustomize/) and the
+[helm-controller](https://fluxcd.io/docs/components/helm/). Kluctl can simply do the same to support Flux.
+
+## Introducing the Kluctl Flux Controller
+An alpha version of the [Kluctl Flux Controller]({{< ref "docs/flux" >}}) has just been released. It allows to
+create [KluctlDeployment]({{< ref "docs/flux/kluctldeployment" >}}) objects which are reconciled in a similar
+fashion as [Kustomizations](https://fluxcd.io/docs/components/kustomize/kustomization/).
+
+Each KluctlDeployment specifies a source object (e.g. a [GitRepository](https://fluxcd.io/docs/components/source/gitrepositories/)),
+the [target]({{< ref "docs/reference/kluctl-project/targets" >}}) to be deployed and some information on how
+to handle kubeconfigs. The controller then regularly reconciles the deployment.
+
+For a simple example, check [this]({{< ref "docs/flux/controller" >}}) documentation.
+
+## Kustomize/Helm vs Kluctl
+If you've already read through the [Kluctl documentation]({{< ref "docs" >}}), you've probably already noticed
+that Kluctl internally uses Kustomize and Helm extensively.
+
+This might raise the question: Why not use plain Kustomize and/or Helm if Flux is already involved? There are multiple
+reasons to prefer Kluctl, which we'll try to elaborate here:
+
+## Kluctl Projects/Deployments
+If you prefer the way Kluctl organizes and structures projects and deployments, then using the Flux Kluctl Controller
+is obviously the best choice. Kluctl allows you to easily glue together what belongs together. If for example, a redis
+database is required to make your application work, you can manage the redis Helm Release and your application in the
+same deployment, including the necessary configuration to let them talk to each other.
+
+To see how different a Kluctl deployment is compared to classic Kustomize/Helm + Flux, you can compare the
+[flux2-kustomize-helm-example](https://github.com/fluxcd/flux2-kustomize-helm-example) and the
+[Kluctl Microservices Demo](https://github.com/kluctl/kluctl-examples/tree/main/microservices-demo/3-templating-and-multi-env)
+([here]({{< ref "docs/guides/tutorials/microservices-demo">}}) is tutorial for the demo).
+
+## Native multi-env support
+Kluctl allows you to natively create deployment projects that can be deployed multiple times to different
+environments/targets. You can for example have one target that is solely meant for `local` (e.g. Kind based) deployments,
+one that targets the `test` environment and one for `prod`. You can then use templating to influence deployments in whatever
+way you like. For example, you could change the `local` target to set all replicas to 1 and omit resource hungry
+support applications.
```

Comment:
support applications for development which you don't want in your production environment or you can affect Ingress domains that correspond to the respective environment.